### ISSUE TITLE
Save Pages

### ISSUE THREAD

**Issue by gacallea** [bbolli/tumblr-utils#99]
@bbolli 

I'm no dev so forgive any misinformed rant (e.g: api doesn't support this), but I would very much appreciate if Pages could be save too. Is that possible to implement?

**Comment by barrycarter** [comment]
If you could tell me how to create a Page (or point me to one), I can take a look to see what's going on and/or add one to my sample blog barrycarter2017.tumblr.com

**Comment by gacallea** [comment]
@barrycarter you can add a page to Tumblr at the bottom of the "edit appearance" option. it looks like this:

<img width="282" alt="screen shot 2017-12-07 at 8 06 47 pm" src="https://user-images.githubusercontent.com/3269984/33733550-65e92794-db8a-11e7-8df0-02298b45ef61.png">

**Comment by barrycarter** [comment]
OK, I found it under "Edit theme", and have now added a page to https://barrycarter2017.tumblr.com/ -- it shows up as a link along with "posts", "likes", "following", etc.

These seem to be fairly static though?

**Comment by gacallea** [comment]
Yes, it is pretty static. I assume that "page" is just another Tumblr special post type that shows up on the menu. Still, it would be nice to be able to get those too.
